docs/platform/concepts/trial-project-features.rst

```diff
@@ -0,0 +1,34 @@
+Trial project features
+======================
+
+Aiven platform offers a free 30 day trial to explore the platform along with many managed Aiven service offerings, including: 
+
+* Apache Cassandra®
+* Apache Kafka®
+* ClickHouse®
+* Grafana®
+* InfluxDB®
+* MySQL
+* OpenSearch®
+* PostgreSQL®
+* Redis®*
+
+
+.. note:: **A credit card is not required to start a trial.** 
+ 
+When you sign up on Aiven for the first time, a default (trial) project will be created. The trial grants full access to services, the management console, and other access including the REST API, CLI tool and Terraform provider. 
+
+**The trial project has some limitations:**
+
+* Auto-generated trial projects cannot be renamed
+* Trial credits cannot be transferred to another project
+* A Maximum of **10** VMs can be created
```

Comment:
should we state more about this limit (the total VM count is the sum of all the nodes on each service cluster)